flake8_pytest_style/visitors.py

```diff
@@ -164,6 +168,22 @@ def _check_parametrize_call(self, node: ast.Call) -> None:
 
         self._check_parametrize_values(node, args.values, multiple_names)
 
+    def _check_raises_call(self, node: ast.Call) -> None:
+        """Checks for all violations regarding `pytest.raises` calls."""
+        args = get_simple_call_args(node)
+        if not args:
+            return
+        exception = args.get_argument('expected_exception', position=0)
+        if not isinstance(exception, ast.Name):
```

Comment:
We should use `get_qualname` to handle cases where the passed argument is a dotted name, e.g. `socket.error` (and also add a test for that)